src/modules/p_usrloc/p_usrloc_mod.h

```diff
@@ -123,5 +123,11 @@ extern int connection_expires;
 extern int alg_location;
 
 extern int  max_loc_nr;
+typedef struct dbm_write {
```

Comment:
Structure name must be a noun.

Reply:
has been changed with "db_shared_param".